frontend/apps/core-app/src/screens/RecipientRegistrationScreen/RecipientRegistrationScreen.component.tsx

```diff
@@ -21,21 +21,35 @@ export const RecipientRegistrationScreenComponent: React.FC<Props> = ({
   onSubmit,
   onCancel,
 }) => {
-  const f = useForm();
+  const f = useForm({ defaultValues: buildDefaultFormValues(forms, records) });
+
+  React.useEffect(() => {
+    f.reset(buildDefaultFormValues(forms, records));
+  }, [JSON.stringify(forms), JSON.stringify(records)]);
 
   return (
     <FormProvider {...f}>
-      <Styles.Container>
-        {forms.map((form, i) => (
-          <Accordion key={form.id} header={form.name}>
-            <RecordEditor form={form} record={records[i]} onChange={() => {}} />
-          </Accordion>
-        ))}
-        <HStack>
-          <Button onPress={onCancel} />
-          <Button onPress={onSubmit} />
-        </HStack>
-      </Styles.Container>
+      <ScrollView width="100%" maxWidth="1180px" marginX="auto">
```

Comment:
where does the 1180 come from, shouldn't this depend on the breakpoints?

Reply:
In the designs the mobile view is full width and the desktop is 1180px. I took this as being full width until you hit 1180.